### PR TITLE
Add up and down votes to the issue templates and contributing guide

### PR DESCRIPTION
**Pull request by jonasrosland**
This PR will add voting details to the feature enhancement request and bug issue templates, as detailed in https://github.com/vmware-tanzu/velero/issues/2574.

Signed-off-by: jonasrosland <jrosland@vmware.com>

### CODE REVIEW

**Comment by carlisia** [comment]
Hey @jonasrosland, run `make update` and update the PR, that'll make ci pass.

File: site/docs/master/start-contributing.md [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
 You may join the Velero community and contribute in many different ways, including helping us design or test new features. For any significant feature we consider adding, we start with a design document. You may find a list of currently in progress new designs here: https://github.com/vmware-tanzu/velero/pulls?q=is%3Aopen+is%3Apr+label%3ADesign. Feel free to review and help us with your input.
 
+You can also vote on issues using :+1: and :-1:, as explained in our [Feature enhancement request][3] and [Bug issue][4] templates. This will help us quantify importance and prioritize issues.
```
Comment (skriss) [inline site/docs/master/start-contributing.md:12]: I would make this same change in `site/docs/v1.4/start-contributing.md` as well, so it's immediately visible in the latest tagged docs.

File: site/docs/master/start-contributing.md [hunk 2]
```diff
@@ -19,3 +21,5 @@ If you are ready to jump in and test, add code, or help with documentation, plea
 
 [1]: https://github.com/vmware-tanzu/velero/blob/master/CODE_OF_CONDUCT.md
 [2]: https://github.com/vmware-tanzu/velero/blob/master/CONTRIBUTING.md
+[3]: https://github.com/vmware-tanzu/velero/blob/master/.github/ISSUE_TEMPLATE/feature-enhancement-request.md
```
Comment (skriss) [inline site/docs/master/start-contributing.md:24]: per prev comment I'd duplicate this change in the v1.4 docs as well, but you'll want to point the links to the v1.4.0 branch (just look at the other links in the v1.4 version of the file)

Comment (jonasrosland) [reply site/docs/master/start-contributing.md:24]: I can do that, but then I'd have to change the issue templates in the v1.4.0 branch as well, correct?

File: site/docs/master/start-contributing.md (hunk 1 above)
Comment (jonasrosland) [reply site/docs/master/start-contributing.md:12]: Adding!

File: site/docs/master/start-contributing.md (hunk 2 above)
Comment (skriss) [reply site/docs/master/start-contributing.md:24]: Ah, you're right. I guess in this case leaving them pointed to `master` makes sense. Next time we generate a new versioned docs site, this'll all work as expected.

Comment (jonasrosland) [reply site/docs/master/start-contributing.md:24]: Perfect!

**Comment by jonasrosland** [comment]
/hold
Need to do a quick fix

**Comment by jonasrosland** [comment]
Release /hold, small bug in go templating fixed :)

**Pull request merged**